### DISCUSSION TITLE
Creating C768 initial conditions for P7?

### DISCUSSION THREAD

**Discussion by benjamin-cash** [ufs-community/ufs-weather-model#1183] [Q&A] [answered]
Hello,

I am looking to make a set of runs using P7 at C768 to complement an existing set of runs I have done at C384. My hope is that we can just run chgres_cube on our existing C384 ICs, but there is a concern it might not handle the fractional grid properly. Is chgres_cube a valid path forward? If not, how would people suggest we go about creating a set of C768 atmospheric ICs for the P7 code? Thanks!

**Comment by pjpegion** [comment, marked as answer]
@benjamin-cash I just shared a google document showing how to use chgres_cube with the fractional grid.
